lib/rex/post/mssql/ui/console.rb

```diff
@@ -126,13 +72,13 @@ def log_error(msg)
           attr_reader :client
 
           # @return [String]
-          attr_accessor :cwd
+          attr_accessor :database_name
```

Comment:
Not a blocker; sometimes it's easier to make this a computed `def database_name` value to ensure there's no edgecases with stale values etc.